### PR TITLE
Sdk 3511

### PR DESCRIPTION
**Pull request by wiperz1789**
Support for Gaze and glasses

### CODE REVIEW

File: python-sdk-samples/affvisionpy-sample.py [hunk 1]
```diff
@@ -160,9 +167,9 @@ def get_command_line_parameters(parser, args):
     frame_width = int(args.res[WIDTH])
     frame_height= int(args.res[HEIGHT])
     return input_file, data, max_num_of_faces, csv_file, output_file, frame_width, frame_height
- 
- 
- 
+
```
Comment (adennie) [inline python-sdk-samples/affvisionpy-sample.py:170]: I think 1 blank line between methods is sufficient, and conforms to PEP 8

File: python-sdk-samples/affvisionpy-sample.py [hunk 2]
```diff
@@ -707,11 +730,14 @@ def write_metrics_to_csv_data_list(args, csv_data, timestamp):
                     current_frame_data["name"] = identity_names_dict[str(identity)]
                 else:
                     current_frame_data["name"] = "Unknown"
-            current_frame_data["confidence"] = round(bounding_box_dict[fid][4],4)
+            current_frame_data["confidence"] = round(bounding_box_dict[fid][4], 4)
+            current_frame_data["gaze_region"] = str(gaze_metric_dict[fid].gaze_region.name)
+            current_frame_data["gaze_confidence"] = str(gaze_metric_dict[fid].confidence)
```
Comment (adennie) [inline python-sdk-samples/affvisionpy-sample.py:735]: shouldn't gaze confidence be rounded, too?

Comment (wiperz1789) [reply python-sdk-samples/affvisionpy-sample.py:735]: It's int value

File: python-sdk-samples/affvisionpy-sample.py (hunk 1 above)
Comment (wiperz1789) [reply python-sdk-samples/affvisionpy-sample.py:168]: Updated to 1 blank line

**Pull request merged**